services/frontend/src/components/searchFilter/SearchFilterView.jsx

```diff
@@ -117,10 +158,33 @@ const SearchFilterView = ({
             mode="multiple"
             maxTagCount={3}
             filterOption={filterOption}
+            aria-label={ariaLabels[2]}
+            aria-autocomplete="list"
+            aria-owns="location_listbox"
+            aria-expanded="false"
+            aria-haspopup="listbox"
+            role="combobox"
+            dropdownRender={(menu) => (
+              <div
+                id="location_listbox"
+                role="listbox"
+                aria-multiselectable="true"
+              >
+                {menu}
+              </div>
+            )}
           >
             {locationOptions.map((value) => {
+              location =
+                value.streetNumber +
+                " " +
+                value.streetName +
+                ", " +
+                value.city +
+                ", " +
+                value.province;
```

Comment:
1. Since you are defining the content of the location, which is the same that is in the Option component, inside of Option, I would just have `{location}` instead of defining the location string twice

2. You need to declare the location variable, and you can concatenate a string like this
```suggestion
              const location = `${value.streetNumber} ${value.streetName}, ${value.city}, ${value.province}`;
```

or if you decompose the object in the param of the map loop, like this `({streetNumber, streetName, city, province, id}) => {...` it could be simplified to the following and you could use straight the variable `id` in the key of the `<Option />` component.

```suggestion
              const location = `${streetNumber} ${streetName}, ${city}, ${province}`;
```

Reply:
Changed it to the second one, it made more sense and easier to follow for users who might update the code later on.